pkgs/development/tools/verible/default.nix

```diff
@@ -31,11 +31,15 @@ buildBazelPackage rec {
     ./remove-unused-deps.patch
   ];
 
-  bazelFlags = [ "--//bazel:use_local_flex_bison" ];
+  bazelFlags = [
+    "--//bazel:use_local_flex_bison"
+    "--javabase=@bazel_tools//tools/jdk:remote_jdk11"
```

Comment:
What are the javabase flags needed for ? This looks like flags that should be in the `build-bazel-package` itself.

I can understand that this might be moved there later in a separate pull request to not affect all bazel projects at once.

But in any case, it might be useful adding a comment describing why this is needed.

Reply:
Is now always needed because of the test-runner @ylecornec?